setup/preamble.tex

```diff
@@ -8,8 +8,9 @@
 \usepackage{textcomp}
 \usepackage{xcolor}
 \usepackage[newfloat, outputdir=out]{minted}
-\setminted[python]{fontsize=\footnotesize, linenos, breaklines, autogobble}
-\setminted[SparQL]{fontsize=\footnotesize, linenos, breaklines, autogobble}
+\setminted[python]{linenos, numbersep=3pt, breaklines, fontsize=\footnotesize}
```

Comment:
Give each of python, sparql, and turtle different styles. lowercase each letter in SparQL.